builder/role_image_test.go

```diff
@@ -455,6 +455,8 @@ func TestBuildRoleImages(t *testing.T) {
 
 	err = roleImageBuilder.BuildRoleImages(
 		rolesManifest.Roles,
+		"test-registry.com:9000",
+		"test-organization",
```

Comment:
We do not seem to have tests for
1. empty registry,
1. empty org,
1. empty registry and org